### PR TITLE
[RHACS] ROX-10956: Update installation instructions to include Admin Credentials link

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s): `rhacs-docs-3.70.0` for right now
[Issue](https://issues.redhat.com/browse/ROX-10956)
Label: **RHACS/next**
[Preview](https://deploy-preview-45775--osdocs.netlify.app/openshift-acs/latest/installing/install-ocp-operator#verify-central-install-operator_install-ocp-operator)

### CODE REVIEW

File: modules/verify-central-install-operator.adoc [hunk 1]
```diff
@@ -28,7 +29,7 @@ $ oc -n stackrox get route central -o jsonpath="{.status.ingress[0].host}"
...
 .. Find the *central* Route and click on the RHACS portal link under the *Location* column.
-. Log in to the RHACS portal using the username *admin* and the autogenerated admin password that you retrieved using the {ocp} CLI in a previous step. Until {product-title} is completely configured (for example, you have the `Central` resource and at least one `SecuredCluster` resource installed and configured), no data is available in the dashboard. The `SecuredCluster` resource can be installed and configured on the same cluster as the `Central` resource. Clusters with the `SecuredCluster` resource are similar to managed clusters in {rh-rhacm-first}.
+. Log in to the RHACS portal using the username *admin* and the  password that you retrieved in a previous step. Until {product-title} is completely configured (for example, you have the `Central` resource and at least one `SecuredCluster` resource installed and configured), no data is available in the dashboard. The `SecuredCluster` resource can be installed and configured on the same cluster as the `Central` resource. Clusters with the `SecuredCluster` resource are similar to managed clusters in {rh-rhacm-first}.
```
Comment (kurlov) [inline modules/verify-central-install-operator.adoc:32]: Nitpick: there is one excess space after `username *admin* and the`

**Review by kurlov — APPROVED** [review]
Looks great!

File: modules/verify-central-install-operator.adoc [hunk 2]
```diff
@@ -12,8 +12,9 @@ After Central finishes installing, log in to the RHACS portal to verify the succ
 . Select the {product-title} Operator from the list of installed Operators.
 . Select the *Central* tab.
 . From the *Centrals* list, select `stackrox-central-services` to view its details.
-. The *Admin Credentials Info* section lists the command you need to run to get the autogenerated password.
-. Use the {ocp} CLI to enter the command to get your admin password:
+. To get the password for the `admin` user, you can either:
+* Click the link provided under **Admin Password Secret Reference**.
```
Comment (lpettyjo) [inline modules/verify-central-install-operator.adoc:16]: You can remove "provided" and just go with "Click the link under..."

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM!

**Comment by kcarmichael08** [comment]
Thank you for the reviews! Should be good to merge now.

**Pull request merged**